packages/button/test/button.test.ts

```diff
@@ -10,10 +10,9 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import '@spectrum-web-components/button/sp-button.js';
-import { Button } from '@spectrum-web-components/button';
-import { html } from 'lit-element';
-import { fixture, elementUpdated, expect } from '@open-wc/testing';
+import '../sp-button.js';
+import { Button } from '..';
+import { html, fixture, elementUpdated, expect } from '@open-wc/testing';
```

Comment:
Was this `html` change just something you noticed while fixing the other imports?